src_files.yml

```diff
@@ -55,7 +55,6 @@ components:
   ]
   files: [
     rtl/components/apb_clkdiv.sv,
-    rtl/components/apb_timer_unit.sv,
```

Comment:
Why did you remove this file?

Reply:
This file has a nearly identical version in the [timer_unit](https://github.com/pulp-platform/timer_unit) dependency. I am unsure which version is newer, and to avoid severe issues I did not delete the file yet, but duplicates can lead to other problems.